calcom/cal.com
# user away check and notify

## What does this PR do?

The PR attempts to complete the requirements asked in the issue :

**As a user**

- I would expect all Event Types to be available but booking them should show info that the Event can't be booked as the user is away.

- A prominent text should also be shown below the user details that the user is currently away

![Screenshot 2022-02-11 at 11 37 09 PM (2)](https://user-images.githubusercontent.com/36269838/153657968-91ae6926-20f9-4b3b-8950-b8a6aac19e38.png)

Fixes # (issue)

https://github.com/calcom/cal.com/issues/1801 - When user is "Away", Booking Page shows no information on why the Event Types aren't available for booking

## Type of change


## How should this be tested?

- Go to https://app.cal.com/event-types
- Using the bottom left User Dropdown, click "Set yourself as away."
- Make sure that you have at least 1 Event Type created.
- Go to your booking page https://cal.com/USERNAME
Notice that only your Avatar and details are there but Event Types are visible and a prominent text appears to indicate the user is away, also if you try to book an event then the error toast pops up indicating away text.



## Checklist:
